### ISSUE TITLE
Master password

### ISSUE THREAD

**Issue by arlukin** [systemconsole/syco#128]
- Investigate ssh-pass/ssh-agent, use that password?
- Use master password in .syco.cnf if it exist. 
  /root/.syco.cnf
  master-password: xxxx  
- Move password file to it's own file.
- Delete master password after 60 minutes with cronjob.
  Check file modify time to determine if 60 minuts has passed.
- Switch in syco that keeps the password for another 60 minutes.
  -k --keep-password -- Do that by touching the file.

**Comment by kribor** [comment]
I guess it would increase the scope somewhat but mounting a ramdisk is just a onliner similar to:
mount -t tmpfs -o size=512m tmpfs /mnt/ramdisk

**Comment by arlukin** [comment]
You are right. It's a pretty good idea. And if we like to, we can have a cronjob unmounting that partion istead of deleting the pasword file from /root/.syco.cnf. This is probably as easy as building the other solution. 
Note to self: Remember how easy it is to mount a ramdisk.
